### DISCUSSION TITLE
Code editor opens twice when hitting .

### DISCUSSION THREAD

**Discussion by simasch** [community/community#9933] [Other Feature Feedback, Questions, & Ideas]
If I hit . in the search result it opens two tabs with the code editor.

**Comment by colinwm** [comment]
The `.` button opens whatever you're looking at in `github.dev`. 

Do you mean that you're getting two github.dev tabs? I can't reproduce this, can you give more details?

**Comment by simasch** [comment]
Yes for example I open
https://cs.github.com/simasch/activemq-artemis/blob/e020090551baf1cff3f374e86d7108aad9f8fb6f/examples/features/standard/rest/push/src/main/java/Order.java#L4

Then I press . and two tabs open:

![image](https://user-images.githubusercontent.com/593352/148985161-583d7973-ad60-4286-94a2-5e4b8a1da632.png)

**Comment by colinwm** [comment]
Interesting, that's definitely a bug. But it isn't happening for me... which browser/OS are you using?

**Reply by simasch** [reply]
Windows 11 with the latest Chrome version
